app/controllers/admin/submissions_controller.rb

```diff
@@ -53,7 +53,7 @@ def create
       @submission =
         SubmissionService.create_submission(
           submission_params.merge(state: 'submitted'),
-          params[:submission][:user_id]
+          submission_params[:user_id]
```

Comment:
@jonallured after we chatted I figured we might as well clean this up too! tried it out locally and worked a-ok